src/pfe/portal/routes/projects/loadtest.route.js

```diff
@@ -48,9 +48,8 @@ router.post('/api/v1/projects/:id/loadtest', validateReq, async function(req,res
     }
 
     try {
-      let runLoadResp = await user.runLoad(project, description);
-      // Response logic completed in ..docker/loadrunner/server.js
-      res.status(runLoadResp.statusCode).send(runLoadResp.body);
+      user.runLoad(project, description);
```

Comment:
Because we aren't awaiting this anymore if `user.runLoad()` throws and error we'll get an unhandled promise rejection warning.
See:
https://github.com/eclipse/codewind/pull/2282/files#diff-196bc19c851941ec243f92f82a372914R185

Reply:
added `then` and `catch` to handle the promise